test/com/jalasoft/search/commons/TestPathHandler.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright (c) 2018 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information").  You shall not disclose such Confidential Information and shall use
+ *   it only in accordance with the terms of the license agreement you entered into with Jalasoft.
+ */
+package com.jalasoft.search.commons;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * <Class description goes here>
```

Comment:
Set the class description here

Reply:
Updated to : 
This Class contains the tests for the methods in PathHandler.